.github/dependabot.yml

```diff
@@ -7,6 +7,13 @@ updates:
     target-branch: 'main'
     open-pull-requests-limit: 10
 
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'monthly'
+    target-branch: '32.x'
+    open-pull-requests-limit: 10
+
```

Comment:
Should 30.x below not be deleted i.e. every branch shifts along one version?

Reply:
Yes; although I also wonder why this is not just *main" branch, and then backport dependency change if appropriate? Like our policy is to always commit to "main" 1st and then backport...

Reply:
That makes sense to me, I will adjust

Reply:
No, read the doc plz

Reply:
As discussed in the PR and ML each branch gets it's own PRs for these updates

Reply:
Thanks Mark, are you perhaps talking about this documentation: https://docs.github.com/en/code-security/getting-started/dependabot-quickstart-guide?  It does make sense to have dependabot running on older branches (e.g. maintenance 31.x) as they might have different dependencies being checked, and the dependabot will prepare the appropriate PRs (I am not sure what a ML is).

So then my original question stands: the archive 30.x should be disabled/removed?  I think so.

Reply:
I apologise for being short previously; dependabot is not configured to check dependencies (apart from security updates by default on the main branch); the idea is to keep the workflows/actions for each branch up-2-date, at some point some of the older branches can be removed from the dependabot.yml config, but we still see commits to branches that have reached EOL when there is a crucial security update